lib/cisco_node_utils/cmd_ref/ntp_config.yaml

```diff
@@ -1,7 +1,13 @@
 # ntp_config
 ---
 source_interface:
-  get_command: "show running-config ntp"
-  get_value: '/^ntp source-interface\s+(.*)$/'
-  set_value: '<state> ntp source-interface <source_interface>'
   default_value: ~
+  nexus:
+    get_command: "show running-config ntp"
+    get_value: '/^ntp source-interface\s+(.*)$/'
+    set_value: '<state> ntp source-interface <source_interface>'
+  ios_xr:
+    context: ["ntp"]
+    get_command: "show running-config ntp"
```

Comment:
If the get_command is the same for XR and nexus you can keep it at the top level rather than duplicating it under each subcategory.